servicemanager/service/smpythonservice.py

```diff
@@ -100,8 +100,8 @@ def _start_from_binary(self):
         makedirs_if_not_exists("logs")
         with open("logs/stdout.txt", "wb") as out, open("logs/stderr.txt", "wb") as err:
             return subprocess.Popen(
-                cmd_with_params[0].split(),
-                shell=False,
+                cmd_with_params[0],
```

Comment:
It would be nice to leave a comment, why it is done this way.

Reply:
the shell flag was to enable the assets_frontend process to run with the same version of python sm was run with which makes it easier to reason about how to start and monitor the process